near-sdk/src/collections/lazy_option.rs

```diff
@@ -9,12 +9,15 @@ use borsh::{to_vec, BorshDeserialize, BorshSerialize};
 
 use crate::env;
 use crate::IntoStorageKey;
+use near_sdk_macros::NearSchema;
 
 const ERR_VALUE_SERIALIZATION: &str = "Cannot serialize value with Borsh";
 const ERR_VALUE_DESERIALIZATION: &str = "Cannot deserialize value with Borsh";
 
 /// An persistent lazy option, that stores a value in the storage.
-#[derive(BorshSerialize, BorshDeserialize)]
+#[derive(BorshSerialize, BorshDeserialize, NearSchema)]
+#[inside_nearsdk]
+#[abi(borsh)]
```

Comment:
1. Do we need to manually derive Borsh/Schema? I would expect that `#[near]` would also work here
2. Instead of `#[inside_nearsdk]`, can we add a parameter, so you can just write `#[near(crate="crate")]`? This way we will also allow re-embedding near_sdk if necessary